src/middleware/environment.js

```diff
@@ -52,9 +52,7 @@ const configureNetwork = (argv, network) => {
       provider = truffleNetwork.provider
     }
   } else if (truffleNetwork.host && truffleNetwork.port) {
-    provider = new Web3.providers.WebsocketProvider(
-      `ws://${truffleNetwork.host}:${truffleNetwork.port}`
-    )
+    provider = new Web3.providers.WebsocketProvider(`${truffleNetwork.host}`)
```

Comment:
Because of this change some commands don't work, e.g. `aragon apm versions`, unless I change my truffle.js to
```diff
- "host": "localhost",
+ "host": "ws://localhost:8545",
  "port": 8545
```
I think we can safely revert this though.

Reply:
Totally. Did this for testing and forgot to revert